SwiftUIHW/SwiftUIHW/ContentView.swift

```diff
@@ -7,21 +7,96 @@
 
 import SwiftUI
 
+/// Домашка с алертом
 struct ContentView: View {
+    
+    @State private var isFirstButtonTapped = false
```

Comment:
```suggestion
    @State private var isAlertShown = false
```
 если флаг что-то показывает, то заканчивается на  Shown